CHANGELOG.md

```diff
@@ -1,5 +1,193 @@
 # Changelog
 
+## 1.8.0
+
+- ### Features
+
+  - **`wrangler dev` - [EverlastingBugstopper], [issue/845] [pull/883]**
+
+    `wrangler dev` is a local proxy server to Cloudflare's preview service, allowing you to automatically re-build and preview your application on `localhost`. This feature is in alpha and we're looking for feedback and bug reports: check out [this issue](https://github.com/cloudflare/wrangler/issues/1047)!
+
+    `wrangler dev` works very similarly to `wrangler preview`, but instead of opening your browser to preview your Worker, it will start a server on `localhost` that will execute your Worker on incoming HTTP requests:
+
+    ```
+    $ wrangler dev
+    ```
+
+    You should be able to send HTTP requests to `localhost:8787`, along with any headers or other request data, and your Worker should execute as expected. Additionally, you'll see `console.log` messages and exceptions appearing in your terminal (!!!).
+
+    For more information on `wrangler dev`'s options, such as passing a custom `host`, `ip`, or `port`, run `wrangler dev` in your terminal for the available flags and options.
+
+    [issue/845]: https://github.com/cloudflare/wrangler/issues/845
+    [pull/883]: https://github.com/cloudflare/wrangler/pull/883
+
+  - **Multi-route support - [ashleymichal], [issue/866] [pull/916]**
+
+    Wrangler now allows developers to publish their Workers projects to multiple routes on a Cloudflare zone.
+
+    To deploy your Workers project to multiple routes, you can migrate from the `route` key to `routes`:
+
+    ```toml
+    name = "worker"
+    type = "javascript"
+    account_id = "youraccountid"
+    # change this line
+    # route = "example.com/foo/*"
+    # to this line
+    routes = ["example.com/foo/*", "example.com/bar/*"]
+    zone_id = "yourzoneid"
+    ```
+
+    [issue/866]: https://github.com/cloudflare/wrangler/issues/866
+    [pull/916]: https://github.com/cloudflare/wrangler/pull/916
+
+  - **`wrangler secret` commands - [ashleymichal], [issue/907] [issue/909] [issue/912] [pull/1045]**
+
+    Wrangler now allows developers to use **secrets** in their Workers codebase. Secrets are encrypted (TODO is that true) values that can be accessed as constants, similar to KV bindings, inside of your Workers code.
```

Comment:
This might be confusing wording because user's might think they then unencrypt them. I'd just say:
```suggestion
    Wrangler now allows developers to use **secrets** in their Workers codebase. Secrets are secure values that can be accessed as constants, similar to text variables, inside of your Workers code.
```